web/ugahacks-7/components/Window/Window.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { ReactElement } from "react";
+import Draggable from "react-draggable";
+import "98.css";
+import styles from "../../styles/Window.module.css";
+
+interface WindowProps {
+  windowTitle: string;
+  width?: string;
+  height?: string;
+  windowType: "browser" | "chat" | "image";
+  showTopBarButtons: boolean;
+  windowHeading?: string;
+  windowBodyText?: string;
+}
+
+const generateWindowStructure = (
+  type: string,
+  windowTitle: string,
+  heading = "",
+  bodyText = "",
+  dataList?: [{ id: number; sender: string; message: string }]
+): ReactElement => {
+  switch (type) {
+    case "browser":
+    default:
+      return (
+        <>
+          <ul className="tree-view">
+            <li className={styles.url}>https://7.ugahacks.com/{windowTitle}</li>
+          </ul>
+          <ul className="tree-view">
+            <li>
+              <h3 className={styles.heading}>{heading}</h3>
+            </li>
+            <li>
+              <p className={styles.bodyText}>{bodyText}</p>
+            </li>
+          </ul>
+        </>
+      );
+    case "chat":
+      return (
+        <>
+          <ul className="tree-view">
+            {dataList &&
+              dataList.map(
+                (data: { id: number; sender: string; message: string }) => (
+                  <li key={data.id}>
+                    <span>{data.sender}</span>: {data.message}
+                  </li>
+                )
+              )}
+          </ul>
+          <div className="field-row-stacked">
+            <textarea></textarea>
+          </div>
+          <button>Send Message</button>
+        </>
+      );
+    case "image":
+      return <p></p>; // Add similar structure to chat layout with mapping over an obj with image data - See Sponsors section on Hacks 6
+  }
+};
+
+const Window = ({
+  windowTitle,
+  width = "50vh",
+  height = "50vh",
```

Comment:
Did we not change this file and set height to auto?

Reply:
I guess we will just solve the merge conflicts in master if we just push this?

Reply:
yeah i think thats a minor thing